### PR TITLE
Add extra chains

### PR DESCRIPTION
**Pull request by ddorgan**
Including validator set example, kovan, foundation and custom spec files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ There are currently three main options which can be used with the parity-deploy
 
 Required:
 
-```--engine``` This option allows you to set the conensus engine of the chain. Currently three methods are supported:
+```--chain``` This option allows you to set the conensus engine of the chain. Currently four methods are supported:
```
Comment (keorn) [inline README.md]: this will get out of sync, no point mentioning the number

**Comment by keorn** [comment]
What is the point of adding the `config/spec/foundation.json` and `config/spec/kovan.json`? These are bundled with Parity: just run `parity --chain foundation`(default) or `parity --chain kovan`.

File: parity-deploy.sh [hunk 2]
```diff
@@ -318,9 +363,38 @@ if [ "$CHAIN_ENGINE" == "tendermint" ] ; then
...
+  build_spec > deployment/chain/spec.json
+  build_docker_config_ethstats
+
+elif [ "$CHAIN_ENGINE" == "kovan" ]  ; then
+
+   cp config/spec/kovan.json deployment/chain/spec.json
+
+elif [ "$CHAIN_ENGINE" == "foundation" ] ; then
+
+   cp config/spec/foundation.json deployment/chain/spec.json
```
Comment (keorn) [inline parity-deploy.sh]: Why is there a need to handle those json specs?

Comment (ddorgan) [inline parity-deploy.sh]: Can we have a quick chat on this. Will just remove these for now and we can speak about the extra chain additions and what should be static (e.g. taking kovan/foundation from parity --chain) and what can be pre-generated (e.g. multi validator set).

**Pull request merged**